plugins/BEdita/Core/src/Job/ServiceRegistry.php

```diff
@@ -49,11 +49,15 @@ public static function get($name)
             return static::$instances[$name];
         }
 
-        $className = Inflector::camelize($name);
-        $fullClassName = App::className($className, 'Job/Service', 'Service');
-        if ($fullClassName === false && strpos($className, '.') === false) {
-            $fullClassName = App::className('BEdita/Core.' . $className, 'Job/Service', 'Service');
+        $plugin = 'BEdita/Core.';
+        if (strpos($name, '.') !== false) {
+            list($plugin, $name) = explode('.', $name);
+            $plugin = Inflector::camelize($plugin) . '.';
         }
+
+        $className = $plugin . Inflector::camelize($name);
+        $fullClassName = App::className($className, 'Job/Service', 'Service');
```

Comment:
This changes the semantics slightly… Using this approach it is impossible to have a service in the app (i.e. not within any plugin), which is not a scenario we are planning, but might be undesirable anyways, especially if and when we will decide to publish `BEdita/Core` and `BEdita/API` as standalone plugins to be re-used in non-standard ways.

I'd probably have changed only:
```diff
- $className = Inflector::camelize($name);
+ $className = implode('.', array_map([Inflector::class, 'camelize'], explode('.', $name)));
```

This should fix the problem without adding this restriction on app services. However, if we decide that app services can be safely excluded, this is good to go.